surveys/2022-12/questions.md

```diff
@@ -0,0 +1,59 @@
+# Survey December 2022
+
+## Personal question
+
+- Which constituencies do you identify with?
+    - Direct end users: Users who run tools themselves
+    - Application operators: Users who interacts with existingm running applications
+    - Application developers: Frontend tools consumers, backend server authors, tools authors
+    - Library & package authors: Users who write librairies and packages to be included on other applications
+    - Node.js core maintainers: Developers working directly on Node.js, individuals participating in Working Groups
+    - Organization with investments in Node.js (eg: Enterprises, Gouvernement bodies, Startups, Non-profit, Education, Security)
+- Are you part of a constituency not covered ?
+    - yes
+    - no
+- If not covered what are you part with ? (open question)
+- Where are you from? (open question or countries list)
+    - country
+-  How long have you been working with Node.js? (numbers list)
+    - 0 - 10+
+- In what kind of organizations are you working in?
+    - big tech
+    - company
+    - startup
+    - university
+    - individual
+    - other
+- Are you part/following some Working Groups? 
+    - yes
+    - no
+- If part/following some Working Groups, which one? (open question)
+
+## Tech questions
+
+- What is important for you ? (Multiple choices)
+    - Good and up to date documentation
+    - Tutorial and examples of core / complexe features
+    - Internationalization of the documentation
+    - Good understanding of the direction of the project
+    - Predictable and stable release
+    - Innovation and new features 
+    - Consistent and intuitive error handling
+    - Runtime diagnostic tooling
+    - Relevant API in core
+    - Good understanding and process for security and CVE in the ecosystem
+    - Loader hook (to allow native typescript support)
+- What is your use case of Node.js ?
+- Do you see others Technical Priorities ? [Technical Priorities](https://github.com/nodejs/node/blob/main/doc/contributing/technical-priorities.md) (open question)
+- Are you using native ESM, e.g. `"type": "module"` or `.mjs` in your Node.js Development?
+- Are you planning to convert your application to native ESM?
+- For those of you wishing to use ESM in an existing application, what have been the pain points or blockers preventing you from doing so (if any)?
+- Any recurrent issue with Node.js ? (open question)
+- What need do you have not covered ? (open question)
+- How do you get your Node.js executables?
+    - With a Node.js version manager: nvm, ns, n, etc.
+    - Using the official installer
+    - Using package managers like npm or yarn and install Node.js as a package
+    - Using a system package manager: apt-get, brew, dnf, yum ...
+    - Other
+- What is your Architecture and OS in which you are running Node.js ? Do you plan to change that in the near future? (open question)
```

Comment:
I think this question would be better as a multiple choice, using the same names from our supported OS/architecture list.

Reply:
Also I would think that lots of people develop in one OS/arch and deploy in another, like develop in Mac and deploy under Linux/Docker.

Reply:
Add the question for both 
- deployed
- developer

Reply:
updated with the two questions. Is the list of architecture supported this one ? Or is there another file containing the list ? https://nodejs.org/api/process.html#processarch 

```
Possible values are: 'arm', 'arm64', 'ia32', 'mips','mipsel', 'ppc', 'ppc64', 's390', 's390x', and 'x64'.
```